lib/views/menu.erb

```diff
@@ -0,0 +1 @@
+Menu page
```

Comment:
Your colleagues who have been working on dishes also have a menu page. Your two groups need to coordinate on these things or else you will just be in a mess with merge conflicts.